dojo/engagement/views.py

```diff
@@ -993,47 +1055,169 @@ def view_risk(request, eid, raid):
     add_findings_form.fields[
         "accepted_findings"].queryset = add_fpage.object_list
 
-    authorized = (request.user == risk_approval.owner.username or request.user.is_staff)
-
-    product_tab = Product_Tab(eng.product.id, title="Risk Exception", tab="engagements")
+    product_tab = Product_Tab(eng.product.id, title="Risk Acceptance", tab="engagements")
     product_tab.setEngagement(eng)
     return render(
-        request, 'dojo/view_risk.html', {
-            'risk_approval': risk_approval,
+        request, 'dojo/view_risk_acceptance.html', {
+            'risk_acceptance': risk_acceptance,
+            'engagement': eng,
             'product_tab': product_tab,
             'accepted_findings': fpage,
-            'notes': risk_approval.notes.all(),
-            'a_file': a_file,
+            'notes': risk_acceptance.notes.all(),
             'eng': eng,
+            'edit_mode': edit_mode,
+            'risk_acceptance_form': risk_acceptance_form,
             'note_form': note_form,
             'replace_form': replace_form,
             'add_findings_form': add_findings_form,
             # 'show_add_findings_form': len(unaccepted_findings),
             'request': request,
             'add_findings': add_fpage,
-            'authorized': authorized,
+            'return_url': get_return_url(request),
         })
 
 
-# @user_passes_test(lambda u: u.is_staff)
-@user_must_be_authorized(Engagement, 'delete', 'eid')
-def delete_risk(request, eid, raid):
-    risk_approval = get_object_or_404(Risk_Acceptance, pk=raid)
+# @user_must_be_authorized(Engagement, 'staff', 'eid')
```

Comment:
why is this commented out?

Reply:
old code that I kept in comments because I wasn't sure if I would need it later or not. it's no longer needed. I will clean it up in the cleanup PR later. for now I want to merge this PR after so many hours :-)